core/src/main/java/org/jdbi/v3/core/argument/internal/PojoPropertyArguments.java

```diff
@@ -45,18 +49,26 @@ protected PojoPropertyArguments(String prefix, Object obj, PojoProperties<?> pro
         super(prefix, obj);
         this.properties = properties;
         this.config = config;
+        names = new HashSet<>();
```

Comment:
can we do this as an expression? names = `properties.getProperties().stream().filter.collect(...)` ?